internal/common/test_configuration.go

```diff
@@ -119,6 +120,9 @@ var DomainResolution = []*models.DomainResolutionResponseDomain{
 		DomainName:    &DomainApps,
 		IPV4Addresses: []strfmt.IPv4{"7.8.9.10/24"},
 		IPV6Addresses: []strfmt.IPv6{"1003:db8::10/120"},
+	},
+	{
+		DomainName: &WildcardDomain,
```

Comment:
Please be more explicit about the lack of response:
```go
	{
		DomainName:    &WildcardDomain,
		IPV4Addresses: []strfmt.IPv4{},
		IPV6Addresses: []strfmt.IPv6{},
	},
```

Reply:
done